test/components/componentTests.ts

```diff
@@ -794,20 +794,50 @@ describe("Component", () => {
 
   describe("restricting rendering through clipPath", () => {
 
-    let c: Plottable.Component;
+    let clippedComponent: Plottable.Component;
     let svg: d3.Selection<void>;
 
     beforeEach(() => {
-      c = new Plottable.Component();
+      clippedComponent = new Plottable.Component();
+      (<any> clippedComponent)._clipPathEnabled = true;
```

Comment:
It's only being used in two tests here. But I concede that it would be cleaner. I'll see if I can do that.